components/TitleBar.js

```diff
@@ -1,8 +1,34 @@
 import styled from "styled-components";
 
-export function TitleBar() {
+import SVGIcon from "@/components/SVGIcon";
+
+export function TitleBar({ onToggle, listView, pathName }) {
```

Comment:
A clean code suggestion:
instead of accepting a prop `pathName` you can accept a boolean prop `showViewToggle` or similar.
This would give the props more semantic meaning and the page which uses the component handles the path logic, which belongs to the page.